lib/Service/VerdictService.php

```diff
@@ -193,6 +182,10 @@ public function tagLastScannedFile(string $localPath, int $fileId): void {
 			$this->tagFile($fileId, TagService::WONT_SCAN);
 			return;
 		}
+		if (!$this->isAllowedByAllowAndBlocklist($localPath)) {
+			$this->tagFile($fileId, TagService::UNSCANNED);
```

Comment:
Not too familiar with the tagging semantics. Only a feeling that `TagService::WONT_SCAN` could be more suited, because you kind of handled the file, but are not allowed to say a verdict. Just imagine a user hitting the "Scan"-Button multiple times, but the "Unscanned" tag never changes. But maybe it's fine, dunno.

Reply:
Strange, there were comments already on this, but I don't see them here. So maybe just delete these, here.